spec/controllers/users_controller_spec.rb

```diff
@@ -85,3 +85,33 @@
     end
   end
 end
+
+RSpec.describe Devise::RegistrationsController, type: :controller do
+  describe '#new' do
+    before(:each) do
+      request.env['devise.mapping'] = Devise.mappings[:user]
+    end
+
+    context 'with sign_up enabled' do
+      render_views
+      it 'works' do
+        CustomSetting['site.signup_enabled'] = '1'
+        get :new
+        expect(response.status).to eq(200)
+        expect(response.body).to include('username', 'biography')
```

Comment:
__[rubocop]__
* Style/MixinGrouping: Put `include` mixins in separate statements. [:ref](https://github.com/bbatsov/ruby-style-guide#mixin-grouping)

<!-- Comment by [SideCI](https://sideci.com) -->